packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -4,11 +4,11 @@ import { FeatureFlags } from '../../../core';
 import * as cxapi from '../../../cx-api';
 import { Construct } from 'constructs';
 import { ApplicationLoadBalancedServiceBase, ApplicationLoadBalancedServiceBaseProps } from '../base/application-load-balanced-service-base';
-import { FargateServiceBaseProps } from '../base/fargate-service-base';
+import { FargateServiceBaseProps, FargateTaskImageOptions } from '../base/fargate-service-base';
 /**
  * The properties for the ApplicationLoadBalancedFargateService service.
  */
-export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationLoadBalancedServiceBaseProps, FargateServiceBaseProps {
+export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationLoadBalancedServiceBaseProps, FargateServiceBaseProps, FargateTaskImageOptions {
```

Comment:
```suggestion
export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationLoadBalancedServiceBaseProps, FargateServiceBaseProps {
```

The `ApplicationLoadBalancedFargateServiceProps` should not extend
TaskImageOptions. You should add a `taskImageOptions` property to this
interface.

Reply:
It is not working actually when I try that it throwing the following error, where taskImageOptions was already existing in 
ApplicationLoadBalancedTaskImageOptions

<img width="1466" alt="image" src="https://user-images.githubusercontent.com/115483524/234377757-9c2c2f34-12cb-4266-a7e9-a9283a043973.png">
<img width="1471" alt="image" src="https://user-images.githubusercontent.com/115483524/234377842-90855a18-5fd4-4bb4-9d3f-c7377705c16c.png">